runtime/v1/project-model/src/main/java/com/walmartlabs/concord/project/yaml/converter/StepConverter.java

```diff
@@ -278,7 +278,7 @@ default Set<VariableMapping> getVarMap(Map<String, Object> options, String key)
             result.add(new VariableMapping(null, sourceExpr, sourceValue, target, true));
         }
 
-        if (key.equals("in") && getWithItems(options) != null) {
+        if ("in".equals(key) && getWithItems(options) != null) {
```

Comment:
Same.

Reply:
The purpose of this change is to be proactively defensive. Though it may be negligibly faster to run this way, it also conforms to a best practice and makes the code easier to read. Also, since the behavior is otherwise equivalent, there is no cost to this change.